### PR TITLE
qt: translate all user-exposed strings

### PR DESCRIPTION
**Pull request by patricklodder**
fixes some missed `tr()` wrappings from the newly created peer management dialogs and debug console screens. Needed to do a fresh set of translations. I haven't found anything else that is not translated in a diff from bitcoin 0.14.1 to current 1.14.4-dev.

### CODE REVIEW

File: src/qt/peerdialog.cpp [hunk 1]
```diff
@@ -165,12 +165,12 @@ void TestPeerDialog::on_testPeer_clicked()
 
     if(!PeerTools::CheckPeerAddress(address))
     {
-        QMessageBox::critical(this, "Test Peer", "Please enter a vaild peer address.", QMessageBox::Ok, QMessageBox::Ok);
+        QMessageBox::critical(this, tr("Test Peer"), tr("Please enter a vaild peer address."), QMessageBox::Ok, QMessageBox::Ok);
```
Comment (chromatic) [inline src/qt/peerdialog.cpp:168]: Typo: valid.

File: src/qt/peerdialog.cpp [hunk 2]
```diff
@@ -118,13 +118,13 @@ void AddPeerDialog::on_addPeer_clicked()
 
     if(!PeerTools::CheckPeerAddress(address))
     {
-        QMessageBox::critical(this, "Add Peer", "Please enter a vaild peer address.", QMessageBox::Ok, QMessageBox::Ok);
+        QMessageBox::critical(this, tr("Add Peer"), tr("Please enter a vaild peer address."), QMessageBox::Ok, QMessageBox::Ok);
```
Comment (chromatic) [inline src/qt/peerdialog.cpp:121]: Typo: valid.

**Comment by alamshafil** [comment]
Sorry I did not realize the typo I made when typing "valid."
Additionally, I can try to translate these new phrases into Spanish in a new pull request. My Spanish is not perfect and help on that pull request would be great.

**Comment by patricklodder** [comment]
No probs I'll fix that 🚀

**Comment by patricklodder** [comment]
Thanks @chromatic for finding some more untranslated strings!

**Review by rnicoll — APPROVED** [review]
ACK; verified messages look fine on the node and private key import dialogues.

**Pull request merged**